src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -77,6 +80,19 @@ private CallbackResponse validateUnavailableDates(CallbackParams callbackParams)
             .build();
     }
 
+    private CallbackResponse validateExperts(CallbackParams callbackParams) {
```

Comment:
There is an example in create claim where we have a ParticularsOfClaimValidator interface. Could we implement that in the two callback handlers to just have one place where this method has to exist?

Reply:
done 👍